pyVor/tests/testPredicates.py

```diff
@@ -122,6 +122,26 @@ def test_incircle_homogeneous(self):
         self.assertEqual(incircle(*inf_triangle[::-1], Point(-1, 1, 1)),
                          -1)
 
+        # Tricky cases:
+        self.assertEqual(incircle(
+            Point(-1, -1, 0), Point(-3, 2, 1), Point(1, 0, 0), Point(0, 1, 0)),
+                         1)
+        self.assertEqual(incircle(
+            Point(-3, 2, 1), Point(-1, -1, 0), Point(1, 0, 0), Point(0, 1, 0)),
+                         -1)
+
+        # The following tests are based on pencil-and-paper drawings
+        # as well as hours spent debugging the DelaunayTriangulation.
+
+        self.assertEqual(incircle(
```

Comment:
Yes, it be equivalent to a lineside test in some cases. Here are some cases, in 2D:
Circle points - query point - result
1 infinite, 2 finite - infinite or not - equivalent to lineside
2 infinite, 1 finite - infinite or not - I honestly don't know. It would be nice if it liked small minimum angles, but it's not terrible if it returns 0 all the time, as far as I can tell. (We don't need to return the faces with infinite vertices, though it would be nice if we could.)
3 infinite - finite - return `ccw(the first three points)`
3 infinite - infinite - doesn't matter, unless we take infinite points as input sometime in the future

We could stand more unit tests for the weirder cases that aren't "Don't care"s, but we now have at least one (the one that fails).